rootfs/bin/doctor

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -eo pipefail
+curl -X POST -d "skip" localhost:8080/doctor
```

Comment:
Actually the API doesn't currently do anything with body data sent to `POST /doctor`. I don't think we need the `-d` option at all in the curl command.